extension-manifest-v2/app/panel/reducers/blocking.js

```diff
@@ -69,6 +70,31 @@ function mergeTrackers(adblockerTrackers, antiTrackingTrackers) {
 			all.set(tracker.name, existing);
 		}
 	});
+	const findTrackerByWhitelistedDomain = domain => Array.from(all.values()).find(
```

Comment:
I wonder if this may become a problem if the whitelist grows. There are three nested loops here:
* For each whitelist entry, it gets called
* Then it iterates over all trackers
* Then it iterates over all tracker domains

I don't see an obvious way to optimize it though.

Extracting `Array.from(all.values())` (also used as the return value) would make sense (I don't know if the JIT can work that out).

Reply:
`Array.from(all.values())` cannot be reused as we modify it with every iteration. I've moved to For-Of loop instead.